<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<?pydev-testrun version="1.0"?><pydev-testsuite>
    <summary errors="1" failures="0" finished="true" name="Test Run: 9" tests="0" total_time="Finished in: 0.00 secs."/>
    <test location="/Users/gabi/git/Rubik/rubik/test/rotateTest.py" status="error" test="rotateTest" time="0">
        <stdout/>
        <stderr><![CDATA[Traceback (most recent call last):
  File "/Users/gabi/.p2/pool/plugins/org.python.pydev.core_9.3.0.202203051235/pysrc/_pydev_runfiles/pydev_runfiles.py", line 468, in __get_module_from_str
    mod = __import__(modname)
  File "/Users/gabi/git/Rubik/rubik/test/rotateTest.py", line 2, in <module>
    import rubik.rotate as rotate
ModuleNotFoundError: No module named 'rubik'
ERROR: Module: rotateTest could not be imported (file: /Users/gabi/git/Rubik/rubik/test/rotateTest.py).
]]></stderr>
    </test>
    <launch mode="run">
        <launch_memento><![CDATA[<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<launchConfiguration local="true" path="Rubik rotateTest.py"/>
]]></launch_memento>
    </launch>
</pydev-testsuite>
